src/share/doc/apidiff.md

```diff
@@ -497,3 +497,18 @@ The tool does _not_ attempt to run `javadoc` locally to generate the page on the
 [jdk.compiler]: https://docs.oracle.com/en/java/javase/17/docs/api/jdk.compiler/module-summary.html
 [Java Language Model API]: https://docs.oracle.com/en/java/javase/17/docs/api/java.compiler/javax/lang/model/package-summary.html
 [Compiler Tree API]: https://docs.oracle.com/en/java/javase/17/docs/api/jdk.compiler/com/sun/source/doctree/package-summary.html
+
+## Example
+
+To compare APIs in the `java.base` module in JDK builds `/local/baseline-jdk` and `/local/updated-jdk`, and place the result in the directory `out`, run the following command:
+
+```
+bin/apidiff --api jdk \
+    --jdk-build=/local/baseline-jdk/build/linux-x86_64-server-fastdebug --jdk-docs docs \
+    --api sv \
+    --jdk-build=/local/updated-jdk/build/linux-x86_64-server-fastdebug --jdk-docs docs \
```

Comment:
`--jdk-docs` is not an api-specific option, and need only be given once, after all the apis have been specified. And, it is only necessary if there are multiple `*docs*` directories in the `build/.../images` directory -- although it is not wrong to give it anyway, just in case.

Stylistically, I would put the first `--api` option on its own line, to better illustrate the similarity between the two different APIs, but that's a matter of personal style.

Reply:
I'm finding all of this very hard to follow, mostly because I have no idea how apidiff is supposed to work. I'm just floundering about in the dark, really.